PubnativeLite/PubnativeLite/MRAID/HyBidMRAIDView.m

```diff
@@ -1176,30 +1176,30 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
 #pragma mark - OM SDK Viewability
 
 - (void)startAdSession {
-    /*
+    
     if (!isAdSessionCreated) {
         adSession = [[HyBidViewabilityManager sharedInstance] createOMIDAdSessionforWebView:currentWebView isVideoAd:NO];
         if (contentInfoView) {
-            [[HyBidViewabilityManager sharedInstance] addFriendlyObstruction:contentInfoView toOMIDAdSession:adSession];
-            [[HyBidViewabilityManager sharedInstance] addFriendlyObstruction:contentInfoViewContainer toOMIDAdSession:adSession];
+            [[HyBidViewabilityManager sharedInstance] addFriendlyObstruction:contentInfoView toOMIDAdSession:adSession withReason:@"This view is related to Content Info" isInterstitial:isInterstitial];
+            [[HyBidViewabilityManager sharedInstance] addFriendlyObstruction:contentInfoViewContainer toOMIDAdSession:adSession withReason:@"This view is related to Content Info" isInterstitial:isInterstitial];
         }
         if (isInterstitial) {
-            [[HyBidViewabilityManager sharedInstance] addFriendlyObstruction:closeEventRegion toOMIDAdSession:adSession];
+            [[HyBidViewabilityManager sharedInstance] addFriendlyObstruction:closeEventRegion toOMIDAdSession:adSession withReason:@"" isInterstitial:isInterstitial];
```

Comment:
you should add the reason explanation for the close button here too, no?

Reply:
We can add but since `OMIDFriendlyObstructionCloseAd`is self explanatory for the purpose, we don't have to. Detailed reason is asked only when purpose is not clear.

```
- (void)addFriendlyObstruction:(UIView *)view toOMIDAdSession:(OMIDPubnativenetAdSession *)omidAdSession withReason:(NSString *)reasonForFriendlyObstruction isInterstitial:(BOOL)isInterstitial {
    if(!self.isViewabilityMeasurementActivated)
        return;
    
    if(omidAdSession != nil){
        NSError *addFriendlyObstructionError;
        if (isInterstitial) {
            [omidAdSession addFriendlyObstruction:view
                                          purpose:OMIDFriendlyObstructionCloseAd
                                   detailedReason:reasonForFriendlyObstruction
                                            error:&addFriendlyObstructionError];
        } else {
            [omidAdSession addFriendlyObstruction:view
                                          purpose:OMIDFriendlyObstructionOther
                                   detailedReason:reasonForFriendlyObstruction
                                            error:&addFriendlyObstructionError];
        }
    }
}
```

Reply:
Yes makes sense 👍